### PR TITLE
add isScannableSize()

### PR DESCRIPTION
**Pull request by butonic**
This PR prevents scanning files bigger than `appConfig->getAvMaxFileSize`. By default it is -1 (no limit).

The title on the settings page seems to never have been visible. I assume because some tipsy or whatever js replaces the input title with the label title. I aligned it with the label, now.

This changes the behavior, because now if you set a File size limit files bigger than it will not even be sent to the scanner. Arguably, that may very well be what you want when reading the label.

Also may need an udpate of the documentation. cc @settermjd 

- [ ] Tests

### CODE REVIEW

**Comment by butonic** [comment]
Ok, the wording for the background scan seems to only have changed in master. in 9.1 it was still `File size limit in bytes, -1 means no limit`

File: lib/avirwrapper.php [hunk 1]
```diff
@@ -136,4 +144,21 @@ private function isWritingMode($mode){
...
+	 * checks the size for webdav PUT requests. defaults to true
+	 * @param $filename
+	 * @return bool
+	 */
+	private function isScannableSize($filename) {
+		$scanSizeLimit = intval($this->appConfig->getAvMaxFileSize());
+		$size = false;
+
+		// PUT via webdav
+		if ($_SERVER['REQUEST_METHOD'] === 'PUT' && isset($_SERVER['CONTENT_LENGTH'])) {
```
Comment (VicDeo) [inline lib/avirwrapper.php]: @butonic would it be testable this way?

Comment (butonic) [inline lib/avirwrapper.php]: Yes, the $_SERVER variable can be set during unit testing and private methods can be made public.

Comment (DeepDiver1975) [inline lib/avirwrapper.php]: Do it. 😆

**Comment by butonic** [comment]
adding sev1 because files cannot be uploaded

**Comment by michaelstingl** [comment]
@butonic @PVince81 Backport to 9.1.x still needed? Or just release with the next (Marketplace) release?

**Comment by michaelstingl** [comment]
@PVince81 It would be cool to have this included in the [next release](https://github.com/owncloud/files_antivirus/milestone/82). I'm not aware of a backport request.

**Comment by PVince81** [comment]
in that case let's not backport. Since this PR is already merged it will be part of the next release of this app in the marketplace

**Comment by VicDeo** [comment]
@PVince81 This is a question that I'd like to have an answer too.
Will we do a marketplace release from a master branch?

**Comment by PVince81** [comment]
@VicDeo master only

**Pull request merged**